Cargo.toml

```diff
@@ -14,7 +14,12 @@ repository = "https://github.com/webrtc-rs/rtp"
 [dependencies]
 byteorder = "1.3.2"
 rand = "0.8.0"
-thiserror = "1.0.23"
+bytes = "1.0.0"
```

Comment:
Looks like this PR overwrote the recent changes with regards to thiserror related error handling refactoring. Could you still keep that changes?

Reply:
Will do that

Reply:
@rainliu not all error values follow pion rtp so unneeded will be removed